packages/amplify-e2e-core/src/init/amplifyPush.ts

```diff
@@ -100,36 +119,45 @@ export function cancelIterativeAmplifyPush(
   return new Promise((resolve, reject) => {
     spawn(getCLIPath(testingWithLatestCodebase), ['push'], { cwd, stripColors: true, noOutputTimeout: pushTimeoutMS })
       .wait('Are you sure you want to continue?')
-      .sendConfirmYes()
+      .sendYes()
       .wait(`Deploying iterative update ${idx.current} of ${idx.max} into`)
       .wait(/.*AWS::AppSync::GraphQLSchema\s*UPDATE_IN_PROGRESS.*/)
       .sendCtrlC()
       .run((err: Error) => {
-        if (err && !/Killed the process as no output receive for/.test(err.message)) {
-          reject(err);
-        } else {
-          resolve();
-        }
+        resolve();
```

Comment:
I would leave a comment here explaining what is being checked. At first glance it's confusing that we are resolving on error.

Could also consider adding a parameter to `runAsync` like `runAsync(expectedErrorPredicate?: (err) => boolean)` that will resolve on an error that matches the predicate

Reply:
the new prompt throws the 130 exit code on CtrlC